### DISCUSSION TITLE
Do @remirror/pm/state browser entries point to the wrong files?

### DISCUSSION THREAD

**Discussion by mtfoley** [remirror/remirror#1386] [Q&A] [answered]
I am trying to update https://github.com/open-sauced/open-sauced to use snowpack instead of webpack.  One of the things I have run into is that I think this package.json file points to the wrong resources for its nested exports of "browser".

https://github.com/remirror/remirror/blob/3d62a9b937e48169fbe8c13871f882bfac74832f/packages/remirror__pm/package.json#L25-L28

This appears to be typical of multiple nested exports, but I think the one above for "browser" should point to "./collab/dist/remirror-pm-collab.browser.esm.js". Right now, these point to files not found in the dist directories installed by NPM.

cc @0-vortex @bdougie

**Comment by 0-vortex** [comment, marked as answer]
`ls -lahH node_modules/@remirror/pm/collab/dist` after installing the package:

<img width="252" alt="Screenshot 2021-11-23 at 21 01 12" src="https://user-images.githubusercontent.com/237133/143087345-11ddb27a-a637-4d6c-8bc9-1790fad89b7b.png">

If no import magic is happening it seems to me that you are right :D
